src/QMCDrivers/QMCDriverNew.h

```diff
@@ -299,6 +299,11 @@ class QMCDriverNew : public QMCDriverInterface, public MPIObjectBase
                                                                     const RealType reserve_walkers,
                                                                     int num_crowds);
 
+  static size_t determineStepsPerBlock(IndexType global_walkers,
```

Comment:
Comment with definition needed.
Can you also make everything const so that it is clear e.g. none of the arguments are changed?

Reply:
It is a static member function and arguments are passed by value. So everything is stateless and const by nature.